Master

bdgscotland opened the pull request:
Allows for individual host checks not only host group checks.
Added support for host groups with no checks attached to them (purely aesthetic)

bdgscotland commented:
Added support for contacts and contact groups into the playbook - they are only configured as part of the original Nagios playbook prior to this.

bcoe commented:
@bdgscotland this is awesome, It's been a while since I've deployed a new nagios server at npm -- we spun it up once and haven't had to touch it, but I'd like to get your changes tested and merged.

I'll try to make time to do this soon, and appreciate the contribution.

bdgscotland commented:
Awesome, @bcoe! Let me know if there are any issues. I have extended it further, but it may be beyond the scope of your role (I'm also still testing it). 

I've added optional WMIC and check_wmi_plus support (for Windows hosts without NRPE or added WMI support).
I've added optional pnp4nagios graphing integration (for graphing).

If you're interested, I can PR it back once I've finished testing.
